ui/ducks/send/send.js

```diff
@@ -570,6 +584,7 @@ export const initialState = {
       value: '0x0',
       gas: '0x0',
       gasPrice: '0x0',
+      type: TRANSACTION_ENVELOPE_TYPES.LEGACY,
```

Comment:
Note to self: Although the intiialState has the params for a legacy transaction, the `updateDraftTransaction` reducer appropriately handles fee market type transactions